crate/server/src/config/command_line/clap_config.rs

```diff
@@ -45,6 +45,7 @@ impl Default for ClapConfig {
             http: HttpConfig::default(),
             proxy: ProxyConfig::default(),
             kms_public_url: None,
+            session_salt: None,
```

Comment:
The session salt should be part of the UIConfig and should not be optional: if the UI is configured, the session salt is mandatory